pkg/executor/nginx/testdata/testRadishConfigWithProxy.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Finnes det et testcase for issuet fra jiraen? 
configurableProxy=false, proxypass definert.